package.json

```diff
@@ -21,20 +21,20 @@
   "author": "Marketing Web Team",
   "license": "ISC",
   "dependencies": {
-    "@babel/cli": "^7.23.0",
-    "@babel/core": "^7.23.2",
-    "@babel/eslint-parser": "^7.22.15",
-    "@babel/preset-env": "^7.23.2",
+    "@babel/cli": "^7.23.4",
+    "@babel/core": "^7.23.5",
+    "@babel/eslint-parser": "^7.23.3",
+    "@babel/preset-env": "^7.23.5",
     "@cypress/webpack-preprocessor": "^5.17.1",
-    "@shelex/cypress-allure-plugin": "^2.40.0",
-    "cypress": "12.14.0",
-    "cypress-real-events": "^1.10.3",
-    "eslint": "^8.51.0",
+    "@shelex/cypress-allure-plugin": "^2.5.0",
```

Comment:
@stevehealy weird, it's picked by npm, so the tag must exists

Reply:
my bad, I read it as 2.50.0. Yes that IS weird.

Reply:
@darkmavis1980 sorry, didn't spot your last comment. I think fixing the critical alert has to come ahead of the test metrics. So lets go with 2.5.0 (if that's what npm audit is happy with), and we can test on our end if there are any negative effects.
Do you want to merge that today? Or save it for Monday?

Reply:
Hey team, if it is okay, may be we can wait until Monday to merge this pr to avoid the possibility of the pipeline breaking for blog. If this is not worth waiting, we can merge, see what happens in the pipeline and go from there. @darkmavis1980 what do you think?